syft/generic/string.py

```diff
@@ -0,0 +1,173 @@
+            self.owner = owner
+
+        str_kwargs = {}
+
+        if object:
+            str_kwargs["object"] = object
+
+        if encoding:
+            str_kwargs["encoding"] = encoding
+
+        if errors:
+            str_kwargs["errors"] = errors
+
+        # Create a str instance as the 'child' attribute
+        self.child = str(**str_kwargs)
+
+        super(String, self).__init__(
+            id=id, owner=self.owner, tags=tags, description=description, child=self.child
+        )
+
+    def send(self, location: BaseWorker):
+        """
+           Sends this String object to the worker specified by 'location'. 
+           and returns a pointer to that string as a StringPointer object.
+
+           Args:
+               location: The BaseWorker object which you want to send this object
+                         to. Note that this is never actually the BaseWorker but instead
+                         a class which inherits the BaseWorker abstraction.
+
+           Returns:
+               A StringPointer objects to self.
+
+        """
+
+        ptr = self.owner.send(self, location)
+
+        return ptr
+
+    @staticmethod
+    def create_pointer(
+        obj,
+        location: BaseWorker = None,
+        id_at_location: (str or int) = None,
+        register: bool = False,
+        owner: BaseWorker = None,
+        ptr_id: (str or int) = None,
+        garbage_collect_data: bool = True,
+    ):
+        """
+           Creates a StringPointer object that points to a String object 'obj'
+           after sending the latter to the worker 'location'.
+
+           Returns:
+                  a StringPointer object
+        """
+
+        if id_at_location is None:
+            id_at_location = obj.id
+
+        if owner is None:
+            owner = obj.owner
+
+        string_pointer = StringPointer(
+            location=location,
+            id_at_location=id_at_location,
+            owner=owner,
+            id=ptr_id,
+            garbage_collect_data=garbage_collect_data,
+        )
+
+        return string_pointer
+
+    @staticmethod
+    def simplify(worker: BaseWorker, string: "String"):
+        """
+           Breaks String object into a tuple of simpler objects, its constituting objects that are
+           serializable.
+
+
+           Args:
+
+               worker: a BaseWorker object
+               string: the String object to be simplified
+
+           Returns:
+               
+               A tuple of simpler objects that are sufficient to recreate
+               a String object that is a clone of `string`.
+
+        """
+
+        def simplify_str(str):
+            return str.encode("utf-8") if str else None
+
+        # Encode the string into a bytes object
+        simple_core = simplify_str(string.child)
+
+        tags = (
+            [simplify_str(tag) if isinstance(tag, str) else tag for tag in string.tags]
+            if string.tags
+            else None
+        )
```

Comment:
can't you do just `tags = sy.serde._simplify(string.tags)`?

Reply:
Done